mkdocs/config/config_options.py

```diff
@@ -647,6 +676,13 @@ def run_validation(self, value):
                     stack.insert(0, frame)
                 tb = ''.join(traceback.format_list(stack))
 
+                if markdown_version >= "3.4":
+                    if isinstance(e, ImportError) or isinstance(e, TypeError):
+                        for word in tb.split():
```

Comment:
`split` isn't robust against punctuation around words, so this probably could be done better without using it. Maybe a `\b(foo|bar)\b` regex.

Actually... seeing as almost all error messages contain quotes, this all code will not even detect this error, as far as I can tell.

```
ImportError: cannot import name 'etree' from 'markdown.util'
```